docs/inputs.md

```diff
@@ -1,4 +1,11 @@
 
+# `autocomplete`
+UI that reveals potential matches to an entered value, ie a value entered via `simple-list`.
+
+### Autocomplete Arguments
```

Comment:
Per [my previous comment](https://github.com/clay/clay-kiln/pull/1080#discussion_r159968270), these are internal arguments to `autocomplete` that aren't exposed as an input for devs to use, so they shouldn't be documented here.